src/shogun/multiclass/BruteKNNSolver.cpp

```diff
@@ -15,7 +15,7 @@ CKNNSolver(k, q, num_classes, min_label, train_labels)
 	nn=NN;
 }
 
-CMulticlassLabels* CBruteKNNSolver::classify_objects(CDistance* distance, const int32_t num_lab, int32_t* train_lab, float64_t* classes) const
+CMulticlassLabels* CBruteKNNSolver::classify_objects(CDistance* knn_distance, const int32_t num_lab, int32_t* train_lab, float64_t* classes) const
```

Comment:
so now that you are doing this kind of changes, and you promised in the past PR of KNN refactor that you gonna do the changes i requested there please do it here:
start using `SGVector` instead of native arrays.